Storybook v5

Quramy opened the pull request:
It closes #7 , #8 . And also it may be related with #6 .

- [x] v4 simple mode
- [x] v4 managed mode
- [x] v5 simple mode
- [x] v5 managed mode

Quramy commented on src/node/browser.ts:
```diff
@@ -97,18 +98,33 @@ export class StorybookBrowser extends Browser {
+          const win = window as ExposedWindow;
+          if (win.__STORYBOOK_CLIENT_API__.raw) {
+            // .raw API exsits only if storybook v5
+            return [win.__STORYBOOK_CLIENT_API__.raw().map(_ => ({ id: _.id, kind: _.kind, story: _.name, version: "v5"  })), null];
```
`id` can be extracted with `raw()` api.

Quramy commented on src/node/browser.ts:
```diff
@@ -275,23 +291,45 @@ $doc.body.appendChild($style);
+            type: "setCurrentStory",
+            args: [
+              {
+                storyId: story.id,
```
With storybook v5, we send `storyId` to storybook channel rather than `kind` and `story` .

Quramy commented:
@heavenshell @sasurau4 how about this?

heavenshell commented:
Thank you for fixing this!
As a quick review of this PR, I think this is good!
My patch is more workaround 😉

I'll checkout this PR and run at my local tomorrow 🙇

Quramy commented:
I've added an example for v5 and managed-mode via 1fcd215 . 

But zisui throws `ScreenshotTimeoutError` ( It seems `emitCapture` does not work)

Reproduce it via:

```sh
cd examples/v5-managed-react
yarn
yarn zisui:all
```

heavenshell commented on src/types.ts:
```diff
@@ -0,0 +1,17 @@
+export type StorybookVersin = "v4" | "v5";
```
`s/StorybookVersin/StorybookVersion/`

Quramy replied:
Ooops 🙇

Quramy commented:
> I've added an example for v5 and managed-mode via [1fcd215](https://github.com/Quramy/zisui/commit/1fcd2158523f6af387a7ecc6da7382acc48945d2) .
> 
> But zisui throws `ScreenshotTimeoutError` ( It seems `emitCapture` does not work)
> 

I fixed it via https://github.com/Quramy/zisui/pull/9/commits/b213a013c96e216ec86854bf0d25d5f4c94efcac#diff-227875b5788e4e39a19edacc3d70eeb0R29

heavenshell commented on src/node/browser.ts:
```diff
@@ -13,16 +13,22 @@ import {
+    if (!kind || Array.isArray(kind) || !story || Array.isArray(story)) return;
+    return `${kind}/${story}`;
+  } else {
+    if (Array.isArray(id)) return;
```
Yeah!! Perfect!!

Quramy commented:
@heavenshell Thanks for your review 😄 

I will self-merge this PR if there are no serious pointed matter from you.

heavenshell commented:
Thank you so much!!!!

sasurau4 commented:
Sorry for late response :bowing_man: 
It's awesome! Thanks for your speedy fixing!!! :tada: :tada: :tada:
